contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsSubscriptions.sol

```diff
@@ -25,6 +25,15 @@ interface IFunctionsSubscriptions {
   // @return subscription - see IFunctionsSubscriptions.Subscription for more information on the structure
   function getSubscription(uint64 subscriptionId) external view returns (Subscription memory);
 
+  // @notice Retrieve details about multiple subscriptions.
+  // @param subscriptionIdStart - the ID of the subscription to start the range at
+  // @param subscriptionIdEnd - the ID of the subscription to end the range at, must be greater than subscriptionIdStart
```

Comment:
In the implementation this is inclusive - let's add that to the comment. Why does End need to be strictly greater than Start? We should allow equal too.